executor/adapter.go

```diff
@@ -239,6 +241,11 @@ func (a *ExecStmt) OriginText() string {
 	return a.Text
 }
 
+// GetHints return the SQL plan hints as a string.
+func (a *ExecStmt) GetHints() string {
+	return hint.RestoreOptimizerHints(plannercore.GenHintsFromPhysicalPlan(a.Plan))
```

Comment:
Does it include hints that not relevant to optimizer? Such as replica read hints, max execution time hints, etc.